plugins/base/src/test/kotlin/translators/DefaultPsiToDocumentableTranslatorTest.kt

```diff
@@ -387,35 +386,176 @@ class DefaultPsiToDocumentableTranslatorTest : BaseAbstractTest() {
     }
 
     @Test
-    fun `should preserve regular functions that look like accessors, but are not accessors`() {
```

Comment:
I realized that the old test didn't make much sense as it was

If the field is public, no getters and setters are set for it (there's a separate test for that). If the field is private, but the getter doesn't qualify, the setter should stay a regular function (added a separate test for that).

So I refactored this one to check specifically for getter lookalikes as it wasn't covered